lib/src/screens/forgot_password_screen.dart

```diff
@@ -0,0 +1,342 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_device_type/flutter_device_type.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import '../constants/constants.dart';
+import '../repositories/user_repository.dart';
+import '../blocs/reset_password_blocs/reset_password_bloc.dart';
+import '../blocs/reset_password_blocs/reset_password_event.dart';
+import '../blocs/reset_password_blocs/reset_password_state.dart';
+
+class ForgotPasswordScreen extends StatefulWidget {
+  const ForgotPasswordScreen({Key? key, required this.userRepository})
+      : super(key: key);
+
+  final UserRepository userRepository;
+
+  @override
+  _ForgotPasswordScreenState createState() => _ForgotPasswordScreenState();
+}
+
+class _ForgotPasswordScreenState extends State<ForgotPasswordScreen> {
+  final TextEditingController emailController = TextEditingController();
+  bool isEmailValid = false;
+
+  final _form = GlobalKey<FormState>();
+  void _saveForm() {
+    final isValid = _form.currentState!.validate();
+    if (!isValid) {
+      return;
+    }
+  }
+
+  void initState() {
+    super.initState();
+    emailController.addListener(onEmailChange);
+  }
+
+  void onEmailChange() {
+    BlocProvider.of<ResetPasswordBloc>(context)
+        .add(EmailChanged(email: emailController.text));
+  }
+
+  void checkResetPassword(context, state) {
+    if (state is EmailChangeSuccess) {
+      isEmailValid = state.isValidEmail;
+    } else if (state is ResetPasswordSubmitSuccess) {
+      if (state.success) {
+        Future.delayed(Duration(seconds: 1), () {
+          Navigator.of(context).pushNamed('/Home');
+          state.success = false;
+        });
+      }
+    }
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    double heightScreen = MediaQuery.of(context).size.height;
+    double widthScreen = MediaQuery.of(context).size.width;
+    CrossAxisAlignment crossAxisAlignment = CrossAxisAlignment.start;
+    MainAxisAlignment mainAxisAlignment = MainAxisAlignment.center;
+    double contextPadding = 0;
+    double iconAndTopSpacing = 0;
+    double iconWidth = 0;
+    double iconHeight = 0;
+    double iconAndTextSpacing = 0;
+    double startTextAndIconSpacing = 0;
+    double containerSpacing = 0;
+    double containerWidth = 0;
+    double containerHeight = 0;
+    double topContainerSpacing = 0;
+    double resetPasswordFontSize = 0;
+    double resetPasswordTextLetterSpacing = 0;
+    double tutorialFontSize = 0;
+    double tutorialLetterSpacing = 0;
+    double tuttorialSpacing = 0;
+    double emailTextSpacing = 0;
+    double enterEmailFieldSpacing = 0;
+    double buttonSpacing = 0;
+    double heightButton = 0;
+    double buttonFontSize = 0;
```

Comment:
why didn't you set this default value for mobile or tab?

Reply:
Because the design of mobile and tablet is the same, space different space. So I set the default value for both mobile and tablet.

Reply:
Because the design of mobile and tablet is the same, space different. So I set the default value for both mobile and tablet.